shaders/src/main/glsl/samples/320es/stable_mergesort.frag

```diff
@@ -25,6 +25,25 @@ uniform vec2 injectionSwitch;
 
 uniform vec2 resolution;
 
+/*
+This shader populates data array with reversed values,
+performs merge sort to get the data to the correct order,
+and then draws an image based on the sorted values and
+the screen coordinates.
+
+The sorting operation is purely integer math, and the
+image drawing uses only linear operations (division, 
+addition) making it numerically stable.
+
+Populating the input data uses an injection switch 
+uniform to avoid the shader compiler from optimizing
+the initialization.
```

Comment:
I would rephase to say that it uses a uniform named injectionSwitch that is guaranteed to be populated with the value (0.0, 1.0) at run-time.

Reply:
Done